certif/app/styles/pages/authenticated/restricted-access.scss

```
@@ -12,7 +12,7 @@
   }
 
   &__opening-title {
-    font-family: $font-open-sans;
+    @extend %pix-title-m;
     font-size: 2rem;
     font-weight: 300;
     letter-spacing: 0.15px;
```

Pour avoir accès à cette page d'accès restreint : https://certif-pr8629.review.pix.org/espace-ferme

On peut supprimer l'ensemble des paramètres (également le line-height) et passer le pix-title en s

```suggestion
    @extend %pix-title-s;
```

---

done